src/mutators.rs

```diff
@@ -127,6 +130,33 @@ pub async fn send_mutations(
         .await?;
 
     println!("Sent mutation: {}", res.status());
-
     Ok(())
 }
+
+#[cfg(test)]
+mod test {
+    use super::create_mutations;
+    use serde::Deserialize;
+    use std::collections::HashMap;
+    use std::fs;
+
+    #[test]
+    fn test_mutators() {
+        let file = fs::read_to_string("mocks/mutators.json").expect("Unable to read file");
+        let data: HashMap<String, Vec<String>> = serde_json::from_str(&file).unwrap();
+
+        for (to_be_mutated, results) in data {
+            let mut muts: Vec<(usize, String)> = vec![];
+            let lines: Vec<&str> = vec![&to_be_mutated];
+            let mut success = false;
+            create_mutations(&lines, &mut muts);
+            for result in results {
+                if muts[0].1 == result {
+                    success = true;
+                }
+            }
+
+            assert_eq!(success, true);
```

Comment:
I think this should be

```suggestion
            let mut success = true;
            create_mutations(&lines, &mut muts);
            for result in results {
                if muts[0].1 != result {
                    success = false;
                }
            }

            assert_eq!(success, true);
```